package.json

```diff
@@ -291,12 +291,12 @@
     "@metamask/address-book-controller": "^6.0.0",
     "@metamask/announcement-controller": "^7.0.0",
     "@metamask/approval-controller": "^7.0.0",
-    "@metamask/assets-controllers": "patch:@metamask/assets-controllers@npm%3A46.0.1#~/.yarn/patches/@metamask-assets-controllers-npm-46.0.1-3c56f3157c.patch",
+    "@metamask/assets-controllers": "patch:@metamask-previews/assets-controllers@47.0.0-preview-33c13522#.yarn/patches/@metamask-assets-controllers-npm-46.0.1-3c56f3157c.patch",
```

Comment:
Preview package with our patch.

TODO - we can cleanup and remove the old unused patches